README.md

```diff
@@ -1,2 +1,79 @@
-# ibm-cloudctl-operator
-This operator provides IBM command line interface to manage cloud resources
+# ibm-platform-api-operator
+
+Operator used to manager IBM Platform API service, a REST service for viewing information about your cluster, manage your cluster, install Helm charts, download CLI tools, and much more.
+
+## Supported platforms
+
+### Platforms
+
+- Red Hat OpenShift Container Plataform 4.x
+
+### Operating Systems
+
+- Red Hat Enterprise Linux CoreOS
+
+## Operator versions
+
+- 3.5.0
+
+## Prerequisites
+
+1. Red Hat OpenShift Container Plataform 4.x installed
```

Comment:
aren't operators also supposed to support OCP 3.11

Reply:
No, operators do not really work on OCP 3.x as there is no Operator Hub. 
For CP4Data, we made a special tar bar build for OCP 3.x with only a few operators